### PR TITLE
Move AzureRG task to PS3

### PR DESCRIPTION
**Pull request by codito**

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/Utility.ps1 [hunk 1]
```diff
@@ -58,6 +58,22 @@ function Write-TaskSpecificTelemetry
...
 }
 
+function Get-AzureCmdletsVersion
+{
+    $module = Get-Module AzureRM -ListAvailable
+    if($module)
+    {
+        return ($module).Version
+    }
+    return (Get-Module Azure -ListAvailable).Version
```
Comment (kmkumaran) [inline Tasks/DeployAzureResourceGroup/Utility.ps1]: I assume you are closing this with @niadak if the check should involve AzureRm.Profile.

Comment (codito) [inline Tasks/DeployAzureResourceGroup/Utility.ps1]: Yes, this is now closed.

File: Tasks/DeployAzureResourceGroup/task.json [hunk 2]
```diff
@@ -208,9 +208,54 @@
     "instanceNameFormat": "Azure Deployment:$(action) action on $(resourceGroupName)",
     "execution": {
         "AzurePowerShell": {
-            "target": "$(currentDirectory)\\DeployAzureResourceGroup.ps1",
+            "target": "$(currentDirectory)\\Legacy\\DeployAzureResourceGroup.ps1",
             "argumentFormat": "",
             "workingDirectory": "$(currentDirectory)"
```
Comment (codito) [inline Tasks/DeployAzureResourceGroup/task.json]: Fix the working directory

File: Tasks/DeployAzureResourceGroup/Legacy/task.json [hunk 3]
```diff
@@ -0,0 +1,216 @@
+{
```
Comment (codito) [inline Tasks/DeployAzureResourceGroup/Legacy/task.json]: Remove

File: Tasks/DeployAzureResourceGroup/Legacy/task.loc.json [hunk 4]
```diff
@@ -0,0 +1,216 @@
+{
```
Comment (codito) [inline Tasks/DeployAzureResourceGroup/Legacy/task.loc.json]: Remove

**Pull request merged**